index.html

```diff
@@ -41,442 +19,942 @@
+                  </li>
+                  <li>
+                    <i class="ion-android-checkmark-circle"></i>First Art event
+                    collaboration 2023
+                  </li>
+                  <li>
+                    <i class="ion-android-checkmark-circle"></i>Official sports
+                    sponsor 2023
+                  </li>
+                </ul>
+              </div>
             </div>
-            <div>
-              <p>Marketing and R$D</p>
-              <p>25%</p>
+          </div>
+        </div>
+      </section>
+      <!-- #about -->
+
+      <!--==========================
+      Call To Action Section
+    ============================-->
+      <section class="call-to-action" class="wow fadeInUp">
+        <div class="container">
+          <div class="row">
+            <div class="col-lg-9 text-center text-lg-left">
+              <h3 class="cta-title">Community</h3>
+              <p class="cta-text">
+                Join project community and be amongst the first to get a free
+                smartstore
+              </p>
+              <div style="max-width: 200px">
+                <div class="social-links d-flex justify-content-around">
+                  <a href="https://twitter.com/turungcommunity" class="twitter"
+                    ><i class="fa fa-twitter fa-2x"></i
+                  ></a>
+                  <a
+                    href="https://facebook.com/Turung-community"
+                    class="facebook"
+                    ><i class="fa fa-facebook fa-2x"></i
+                  ></a>
+
+                  <a href="https://linkedin.com/company/turung" class="linkedin"
+                    ><i class="fa fa-linkedin fa-2x"></i
+                  ></a>
+                  <a href="https://t.me/turungcommunity" class="telegram"
+                    ><i class="fa fa-telegram fa-2x"></i
+                  ></a>
+                </div>
+              </div>
             </div>
-            <div>
-              <p>Reserve</p>
-              <p>10%</p>
+            <div class="col-lg-3 cta-btn-container text-center">
+              <a
+                class="cta-btn align-middle"
+                target="_blank"
+                href="./assets/files/turung_whitepaper.pdf"
+                >Whitepaper</a
+              >
             </div>
-            <div>
-              <p>Network</p>
-              <p>10%</p>
+          </div>
+          <div class=""></div>
+        </div>
+      </section>
+      <!-- #call-to-action -->
+
+      <!--==========================
+      Clients Section
+    ============================-->
+      <section id="clients" class="wow fadeInUp contributors">
+        <div class="container">
+          <div class="section-header">
+            <h2>Contributors</h2>
+          </div>
+
+          <div class="owl-carousel clients-carousel">
+            <div class="member mx-3">
+              <div class="pic">
+                <img
```

Comment:
![image](https://user-images.githubusercontent.com/60205816/179097137-b7c35934-bfa3-417a-ac62-6ad2f8762b42.png)

The contributors' images on mobile view are squashed, I suggest you use one image on mobile view instead of two.

Reply:
adjusted slider element